includeDaniel/fibonacci
# Enable users to import with import/require

# Description

The users right now can't use `"commonjs"/require` with your package because you're using `type="module"` witch obligates all the users that download your package to use `"module/import"` witch is a issue, because in legacy projects you might need still use `"commonjs"/require`

What I changed

- I removed `type="module"` from your `package.json` (with this change the users can use `import/require syntax` with your package)
- I renamed rollup.config`.js` and jest.config`.js` files to rollup.config`.mjs`, jest.config`.mjs` (with this change you can still use `import syntax` inside those files)

# Testing

- Setup 
    - Create a empty node app with `npm init`
    - Generate a new `/dist` with the command `yarn build` and copy `/dist` folder inside a `node_modules/@includedaniel/fibonacci` folder

- Testing with `commonjs/require syntax`
    - Create a index.js file with the following content
       ```js
        const fibonacci = require('@includedaniel/fibonacci')
        console.log(fibonacci(25))
        ```
    - With the command `Node index.js` you should get `75025`
    
- Testing with `module/import syntax`
    - Create a index.mjs file with the following content (if you want use index.js file you should add `type="module"` at this node app package.json)
       ```js
        import fibonacci from '@includedaniel/fibonacci'
        console.log(fibonacci(25))
        ```
    - With the command `Node index.mjs` you should get `75025`
